corehq/apps/app_manager/management/commands/build_apps.py

```diff
@@ -62,6 +62,8 @@ def handle(self, *args, **options):
                     app = RemoteApp.wrap(j)
 
             app.version = 1
+            if not app.domain:
+                app.domain = "test"
```

Comment:
`domain` is needed for testing privileges. Unfortunately with this change all privilege checks will return False but at least they won't fail.